### ISSUE TITLE
App after build does not start.

### ISSUE THREAD

**Issue by heykulthe** [serkandyck/flutter_vpn_example#12]
I cloned your repo as it is and then built it after updating all the dependencies. The build was successful, but when I installed the app, the app stops working ( I mean it doesn't even start)
I used the regular command: `flutter build apk`
Can you help me with this?

**Comment by serkandyck** [comment]
any error output ?

**Comment by heykulthe** [comment]
No, the build is successful as usual. But, when I start the app, it stops working. It doesn't even display anything. There is a just a notification saying 'The app has stopped working.' I tried running `flutter doctor`, but even that doesn't give any errors.

The app is successfully installed (although, I get a prompt asking it's not protected by google play but I choose the 'Install anyway' option) but when I start it, I don't know what goes wrong.

![Screenshot_20200701-070609](https://user-images.githubusercontent.com/55309498/86193545-c9cf7d00-bb69-11ea-8c58-d4305f55b33a.png)
![Screenshot_20200701-070739](https://user-images.githubusercontent.com/55309498/86193548-cb994080-bb69-11ea-954b-4267746d65c6.png)

**Comment by heykulthe** [comment]
Well, one thing I would like to check into is that... what is this abiFilter and what do I have to do with it?
(Do, I have to input any code for it? Or change anything in build.gradle , etc.)
Please mention in detail.

**Comment by serkandyck** [comment]
i'm using this library, https://github.com/X-dea/Flutter-VPN

**Comment by heykulthe** [comment]
Well, I am using the same library as well. I have updated all the dependencies, used `flutter pub get ` But can't figure it out.
Well, one thing to ask that can letting everything as default, will this app run on Android 6.0.1 (the one I used for testing) or do I need to tweak somewhere in the  build files?

**Comment by Deepankar1993** [comment]
I am facing the same issue

**Comment by heykulthe** [comment]
@serkandyck, any idea on how to resolve it?

**Comment by msngithub** [comment]
I am facing the same issue
